### ISSUE TITLE
Make a contribution failied

### ISSUE THREAD

**Issue by bposh** [sergi/tabgrenade#44]
Sergi - I like your add-on so I wanted to kick in.  I tried to make a contribution 2 ways - I wasn't successful either way.

1) I used the make a contribution button - left the selector on A one-time suggested contribution of $1.99, made a comment, then hit- make a contribution.  The frame disappeared, no popup for additional input or anything.

Ok - so, maybe I'm supposed to pick PayPal (which is how I wanted to pay) so: 

2)  I used the make a contribution button - left the selector on A one-time suggested contribution of $1.99, then selected  PayPal,  This then took me to PayPal where I logged in - but it left me at an entry page with no indicator of who to pay - I didn't know how to pay you? 

Bottom line is that I wanted to send you a couple of buck and this is way more effort than I wanted to make.  Tell me an easy way I can make a contribution and I'll try one more time.  I do love using Tab Grenade
Thanks 
BPosh
